api/lightning_utils.py

```diff
@@ -0,0 +1,189 @@
+import requests, json
```

Comment:
In my mind, something named "utils" would not touch models and the DB, so it can be quickly imported from a shell to test out things without breaking anything. Stuff touching the DB would come in methods on the model classes...

Reply:
Actually I don't think these are really "utils". I think we need a class that deals with Alby, and that can be mocked during tests as well, otherwise we will be unable to test this.

See `MempoolSpaceBTCClient` versus `MockBTCClient`... and `get_btc_client`

Basically, we should have something similar... AlbyLightningClient versus MockLightningClient... and if we run tests, it automatically6 uses the mock version and it returns dummy data (looks up orders that exist in the DB and returns payments for them). So we will be able to test, and it will just work without hitting Alby... Also, if we have it as a class, we can have something other than Alby as well, as long as it respects the same method signatures.

Reply:
Done, with empty Mock methods for now.